src/db/study.ts

```diff
@@ -114,39 +114,34 @@ const normalizeAllowedUsers = (
         }
   })
 
-export const getMainStudy = async (organizationId: string) => {
-  const study = await prismaClient.study.findFirst({
-    where: { organizationId },
+export const getOrganizationStudiesOrderedByStartDate = async (organizationId: string) => {
+  const studies = await prismaClient.study.findMany({
+    where: {
+      organizationId,
+    },
     include: fullStudyInclude,
     orderBy: { startDate: 'desc' },
   })
-  if (!study) {
-    return null
-  }
-  return { ...study, allowedUsers: normalizeAllowedUsers(study.allowedUsers, study.level, organizationId) }
+  return studies.map((study) => ({
+    ...study,
+    allowedUsers: normalizeAllowedUsers(study.allowedUsers, study.level, organizationId),
+  }))
 }
 
 export const getStudiesByUser = async (user: User) => {
   const userOrganizations = await getUserOrganizations(user.email)
 
   // Be carefull: study on this query is shown to a lot of user
   // Never display sensitive data here (like emission source)
-  return prismaClient.study.findMany({
-    where: {
-      OR: [
-        { organizationId: { in: userOrganizations.map((organization) => organization.id) } },
-        { allowedUsers: { some: { userId: user.id } } },
-        { contributors: { some: { userId: user.id } } },
-      ],
-    },
+  return await prismaClient.study.findMany({
+    where: { organizationId: { in: userOrganizations.map((organization) => organization.id) } },
   })
 }
 
 export const getStudiesByUserAndOrganization = async (user: User, organizationId: string) => {
```

Comment:
user n'est plus utilisé
```suggestion
export const getStudiesByUserAndOrganization = async (organizationId: string) => {
```